onnxmltools/utils/float16_converter.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+from struct import unpack
+from ..convert.common.data_types import *
+from onnxmltools.utils import load_model, save_model
+
+
+def npfloat16_to_int(np_list):
```

Comment:
Please change `npfloat16_to_int` to `_npfloat16_to_int` if it's an internal function. I know often I also forgot doing so but let's keep this as restrictive as possible. Thanks a lot.

Reply:
will do